internal/proto/data_coord.proto

```diff
@@ -167,7 +167,7 @@ message VchannelInfo {
   internal.MsgPosition seek_position = 3;
   repeated SegmentInfo unflushedSegments = 4;
   repeated SegmentInfo flushedSegments = 5;
-  repeated int64 dropped_segments = 6;
+  repeated SegmentInfo dropped_segments = 6;
 }
 
```

Comment:
Dropped segments field is used in datanode. Merging this will make master branch compilation fails. Could you please rebase and fix related usage?

Reply:
okay